File: src/main/java/seremis/geninfusion/soul/gene/GeneUseOldAI.java
package seremis.geninfusion.soul.gene;

import seremis.geninfusion.api.soul.IAllele;
import seremis.geninfusion.api.soul.SoulHelper;
import seremis.geninfusion.api.soul.lib.Genes;
import seremis.geninfusion.soul.MasterGene;
import seremis.geninfusion.soul.allele.AlleleBoolean;

public class GeneUseOldAI extends MasterGene {

    public GeneUseOldAI() {
        addControlledGene(Genes.GENE_CEASE_AI_MOVEMENT);
    }

    @Override
    public Class<? extends IAllele> possibleAlleles() {
        return AlleleBoolean.class;
    }
}

File: src/main/java/seremis/geninfusion/soul/gene/GeneUseOldAI.scala
package seremis.geninfusion.soul.gene

import seremis.geninfusion.api.soul.EnumAlleleType
import seremis.geninfusion.api.soul.lib.Genes
import seremis.geninfusion.soul.MasterGene

class GeneUseOldAI extends MasterGene(EnumAlleleType.BOOLEAN) {
    addControlledGene(Genes.GENE_CEASE_AI_MOVEMENT)
}
